foodkept/Pages/User/Register.cshtml.cs

```diff
@@ -142,7 +142,7 @@ public async Task<IActionResult> OnGetConfirmEmailAsync(string token, string ema
                 return RedirectToPage("./Register");
             var result = await userManager.ConfirmEmailAsync(user, token);
             await signInManager.SignInAsync(user, false);
-            return RedirectToPage("./index");
+            return RedirectToPage("../index");
```

Comment:
What is the difference between . and .. ?

Reply:
'..' goes 1 directory back, '.' - current directory. But I think there is no difference in this case, so it can be removed.